purview/utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+from purview.utils.teiid import Teiid
+
+__all__ = ['Teiid']
```

Comment:
Can you please remove this? We'll hopefully never do `from utils import *` and it's a pain to maintain.